challenge5/index.html

```diff
@@ -0,0 +1,292 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link href="https://fonts.googleapis.com/css2?family=Mulish:wght@400;500&display=swap" rel="stylesheet">
+  <script src="https://cdn.tailwindcss.com"></script>
+  <title>Document</title>
+  <script>
+    tailwind.config = {
+      theme: {
+        extend: {
+          colors: {
+            primary_color: "#FFFFFF",
+            secondary_color: "#458FF6",
+            top_gradient: "#5A98F2",
+            bottom_gradient: "#67C3F3",
+            background: "#FAF9F6",
+          },
+          lineHeight: {
+            12: "60px",
+          },
```

Comment:
Is this being used? If not we can delete it

If we are using it, in tailwind `leading` is used for line height, and a `leading-10` is `40px`, so `60px` should be a `leading-15`